tests/test_utils.py

```diff
@@ -50,9 +49,7 @@ def test_address_valid(value, name, email):
     '\u001f@example.com',
     '"@example.com',
     '\"@example.com',
-    '`@example.com',
     ',@example.com',
-    'foobar <foobar`@example.com>',
```

Comment:
surely this case should still fail? If not can you add a fail with with `Name <email@address>` format.